src/app/common/stores/applications/SsvWeb/Wallet.store.ts

```diff
@@ -227,15 +232,15 @@ class WalletStore extends BaseStore implements Wallet {
   /**
    * Fetch user balances and fees
    */
-  async balanceHandler(balance: any) {
+  async onBalanceChangeCallback(balance: any) {
     if (balance) await this.initializeUserInfo();
   }
 
   /**
    * User Network handler
    * @param networkId: any
    */
-  async networkHandler(networkId: any) {
+  async onNetworkChangeCallback(networkId: any) {
     this.networkId = networkId;
     if (networkId !== 5 && networkId !== undefined) {
```

Comment:
Move network ID 5 to constants and name that constant according to the network name (goerli, prater, whatever it is)

Reply:
done